### ISSUE TITLE
Error downloading sdk from starter page.

### ISSUE THREAD

**Issue by slow1ne** [prominic/Moonshine-IDE#563]
- [ ] Review the documentation: https://github.com/prominic/Moonshine-IDE
- [ ] Search for existing issues: https://github.com/prominic/Moonshine-IDE/issues
- [ ] Use the latest stable version at: http://moonshine-ide.com
- [ ] Let us know how to reproduce the issue. Include a code sample, or share a project that reproduces the issue

## Environment

Windows 7 professional 32bit
## Description

I have Apache-royale installed via npm, but it does not know this.
I also have flex 2, 3.2, 3.6a sdk as well. 
## Steps to Reproduce

just opening Moonshine
## Expected Behavior

Download the software from the starter page.
## Actual Behavior/Errors
Error messages instead of downloading.
### Errors
Error #3219
Error: Error #3219
	at flash.desktop::NativeProcess/start()
	at actionScripts.utils::MSDKIdownloadUtil/runAppStoreHelperWindows()
	at actionScripts.utils::MSDKIdownloadUtil/updateDescriptorLoader_completeHandler()
	at flash.events::EventDispatcher/dispatchEvent()
	at flash.net::URLLoader/onComplete()

Error #3219
Error: Error #3219
	at flash.desktop::NativeProcess/start()
	at actionScripts.utils::MSDKIdownloadUtil/runAppStoreHelperWindows()
	at actionScripts.utils::MSDKIdownloadUtil/runOrDownloadSDKInstaller()
	at components.popup::GettingStartedPopup/runAppStoreHelper()
	at components.popup::GettingStartedPopup/__viewHelper_DOWNLOAD_COMPONENT()
	at flash.events::EventDispatcher/dispatchEvent()
	at mx.core::UIComponent/dispatchEvent()
	at components.renderers::PackageDependencyRenderer/onDownloadRequest()
	at components.renderers::PackageDependencyRenderer/__imgDownload_click()

Error #3219
Error: Error #3219
	at flash.desktop::NativeProcess/start()
	at actionScripts.utils::MSDKIdownloadUtil/runAppStoreHelperWindows()
	at actionScripts.utils::MSDKIdownloadUtil/runOrDownloadSDKInstaller()
	at components.popup::GettingStartedPopup/runAppStoreHelper()
	at components.popup::GettingStartedPopup/__viewHelper_DOWNLOAD_COMPONENT()
	at flash.events::EventDispatcher/dispatchEvent()
	at mx.core::UIComponent/dispatchEvent()
	at components.renderers::PackageDependencyRenderer/onDownloadRequest()
	at components.renderers::PackageDependencyRenderer/__imgDownload_click()

Error #3219
Error: Error #3219
	at flash.desktop::NativeProcess/start()
	at actionScripts.utils::MSDKIdownloadUtil/runAppStoreHelperWindows()
	at actionScripts.utils::MSDKIdownloadUtil/runOrDownloadSDKInstaller()
	at components.popup::GettingStartedPopup/runAppStoreHelper()
	at components.popup::GettingStartedPopup/___GettingStartedPopup_Button1_click()

**Comment by piotrzarzycki21** [comment]
@slow1ne can you test using [development version](https://moonshine-ide.com/downloads/releases/dev/windows/) so we could know whether newest release is triggering same issue.

**Comment by rat-moonshine** [comment]
The last stable version should work for the purpose, as they were meant for distribution. I'll give this a test, however I'm not sure how sooner I can able to get a Windows 7 test environment for this.

**Comment by rat-moonshine** [comment]
I gave some tests on Windows 10 (as this only available to me now) and I didn't able to reproduce any error with last stable (v2.2.0) version:

1. Removed any existing application data from the system
2. Removed any existing Moonshine SDK Installer from the system
3. Removed c:\MoonshineSDKs folder from the system
4. Removed any existing Moonshine application from the system
5. Installed v2.2.0 downloaded from Moonshine website
6. As expected, it opened me the 'Getting Started' tab upon start of the Moonshine pointing many SDKs that I'm missing
7. Clicking on any download icon starts the downloading process of Moonshine SDK Installer distribution version from the Moonshine website
![image](https://user-images.githubusercontent.com/24370870/58408417-6122f080-808b-11e9-8037-ac41f0963414.png)
8. Upon download Moonshine SDK Installer auto-started
9. Closed the Moonshine SDK Installer application and tried to open by clicking on any download link in 'Getting Started' tab again - did this multiple times, and even after restarting Moonshine, no error thrown and Moonshine SDK Installer opened each time without any fail so far

```
Error: Error #3219
at flash.desktop::NativeProcess/start()
at actionScripts.utils::MSDKIdownloadUtil/runAppStoreHelperWindows()
```
This above error is suppose to run the downloaded/installed Moonshine SDK Installer application. So far I didn't found any problem during my tests. I'll see if I can arrange an Windows 7 environment for test.

**Comment by piotrzarzycki21** [comment]
@rat-moonshine Even if @slow1ne try right now development version he won't have the newest SDK Installer with it am I right ? Cause it's not on the website ? Can we have for dev version uploaded to the website somehow or at least in the future work on that.

**Comment by rat-moonshine** [comment]
> 
> 
> @rat-moonshine Even if @slow1ne try right now development version he won't have the newest SDK Installer with it am I right ? Cause it's not on the website ? Can we have for dev version uploaded to the website somehow or at least in the future work on that.

That is right, new build of SDK Installer will not available until they also uploaded to the Moonshine website. We already have a flag to determine if a development or production version in Moonshine, so it wouldn't be hard to direct to a different link to download SDK Installer for development builds.

**Comment by piotrzarzycki21** [comment]
> That is right, new build of SDK Installer will not available until they also uploaded to the Moonshine website. We already have a flag to determine if a development or production version in Moonshine, so it wouldn't be hard to direct to a different link to download SDK Installer for development builds.

We should definitely do this, cause I'm catching myself all the time that I would like to test something and I have simply an old SDK Installer.

**Comment by piotrzarzycki21** [comment]
@slow1ne please try our newest release and let us know whether you experience the same problems.
